csrc/mmdeploy/apis/c/mmdeploy/classifier.cpp

```diff
@@ -40,6 +40,18 @@ int mmdeploy_classifier_create_by_path(const char* model_path, const char* devic
   return ec;
 }
 
+int mmdeploy_classifier_create_by_buffer(const void* buffer, int size, const char* device_name,
```

Comment:
We don't suggest creating another API. It's kinda redundant because `mmdeploy_model_create` and `mmdeploy_classifier_create` are enough to make up for it